CONTRIBUTING.md

```diff
@@ -66,6 +66,16 @@ creating a build file :
 - The output of the files should be placed in directory `bin` relative to the local project directory.
 - All the bin directories (including object files) are to be ignored in the repo.
 
+#### Final Directory Structure
```

Comment:
I've also added a final directory structure, should be useful for someone who is knowledgeable working with C projects and wants to skip to the end.